prow/plugins/lgtm/lgtm_test.go

```diff
@@ -229,24 +230,165 @@ func TestLGTMComment(t *testing.T) {
 	}
 }
 
+func TestLGTMCommentWithLGTMNoti(t *testing.T) {
+	// "a" is the author, "a", "r1", and "r2" are reviewers.
```

Comment:
go ahead and just change the string literals instead of this

Reply:
yes please, or at least use longer names, I shouldn't need to reference this to read the test cases :-)

Reply:
👍 I will make changes to other tests also